api/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:8.2.9-apache-bullseye as base
+FROM php:8.3.9-apache-bullseye as base
```

Comment:
Do we want to stick with this path release? It looks like they are currently on `8.3.13`.

https://www.php.net/releases/

Reply:
I chose 8.2.9 because that matches the PHP version in our Azure app service image.